src/app/thankYou/summary/thankYouSummary.component.js

```diff
@@ -73,7 +74,9 @@ class ThankYouSummaryController {
 
         this.analyticsFactory.pageLoaded()
         this.analyticsFactory.setPurchaseNumber(data.rawData['purchase-number'])
-        this.analyticsFactory.transactionEvent(this.purchase)
+        if (!this.envService.read('isBrandedCheckout')) {
+          this.analyticsFactory.transactionEvent(this.purchase)
```

Comment:
I wonder how this change impacts the two thank you scenarios in branded checkout:
1) The default thank you page
2) A custom thank you page

Is there any easy way to confirm this?

Reply:
`this.brandedAnalyticsFactory.purchase()` is still always called by https://github.com/CruGlobal/give-web/blob/baa8293d78c595cee8d9eaafcd77696488c29ad2/src/app/thankYou/summary/thankYouSummary.component.js#L71-L73 This just changes it to stop emitting another `purchase` event from `analyticsFactory`. I tested the before and after on JPF, which has a custom thank you page. Let me know if I'm missing something though.

Reply:
Good point. Just tested on josh.org, which doesn't have a custom thank you page, and there is only one `purchase` event, as expected.